include/dxc/HLSL/DxilPipelineStateValidation.h

```diff
@@ -198,7 +198,7 @@ struct PSVSemanticIndexes {
   uint32_t Offset;
   PSVSemanticIndexes() : Offset(0) {}
   PSVSemanticIndexes(uint32_t offset) : Offset(offset) {}
-  uint32_t *Get(const PSVSemanticIndexTable &table) const { table.Get(Offset); }
+  const uint32_t *Get(const PSVSemanticIndexTable &table) const { return table.Get(Offset); }
```

Comment:
`const` here is redundant.

Reply:
the underlying table.Get returns a `const uint32_t *`
you'd get a compile error without the const.

Reply:
Ah, sorry. Didn't notice the `*`. My bad.